pkg/utils/utils.go

```diff
@@ -296,7 +297,7 @@ type ConntrackVal struct {
 
 func ConvConntrackV6ToByte(key ConntrackKeyV6) []byte {
 	ipSize := unsafe.Sizeof(key)
-	byteArray := (*[38]byte)(unsafe.Pointer(&key))
+	byteArray := (*[40]byte)(unsafe.Pointer(&key))
```

Comment:
could we please doc how we got to 40? :)

Reply:
Also do we need protection on length of stuff we copy in line 308 as well?

Reply:
actually, it is the size of the v6 contract key..I will update it.